content-api-post.js

```diff
@@ -27,7 +27,7 @@ exports.handler = function (event, context, callback) {
   const data = JSON.parse(event.body);
   const decodedImage = Buffer.from(data.body, 'base64');
 
-  if (!decodedImage || !data.accountId || !data.type) {
+  if (!decodedImage || !(data.accountId && data.championFundId) || !data.type) {
```

Comment:
@tbg-jasonfung this still doesn't look right to me. Think about the case where there is an account ID but no fund. Parenthesised like this you're still requiring both.